src/main/java/org/folio/services/consortium/ServicePointSynchronizationVerticle.java

```diff
@@ -0,0 +1,111 @@
+package org.folio.services.consortium;
+
+import static org.folio.rest.tools.utils.ModuleName.getModuleName;
+import static org.folio.rest.tools.utils.ModuleName.getModuleVersion;
+import static org.folio.services.domainevent.ServicePointEventType.SERVICE_POINT_CREATED;
+import static org.folio.services.domainevent.ServicePointEventType.SERVICE_POINT_DELETED;
+import static org.folio.services.domainevent.ServicePointEventType.SERVICE_POINT_UPDATED;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.http.HttpClient;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.kafka.AsyncRecordHandler;
+import org.folio.kafka.GlobalLoadSensor;
+import org.folio.kafka.KafkaConfig;
+import org.folio.kafka.KafkaConsumerWrapper;
+import org.folio.kafka.SubscriptionDefinition;
+import org.folio.kafka.services.KafkaEnvironmentProperties;
+import org.folio.kafka.services.KafkaTopic;
+import org.folio.services.caches.ConsortiumDataCache;
+import org.folio.services.consortium.handler.ServicePointSynchronizationCreateHandler;
+import org.folio.services.consortium.handler.ServicePointSynchronizationDeleteHandler;
+import org.folio.services.consortium.handler.ServicePointSynchronizationUpdateHandler;
+import org.folio.services.domainevent.ServicePointEventType;
+
+public class ServicePointSynchronizationVerticle extends AbstractVerticle {
```

Comment:
There is already existing `SynchronizationVerticle` that would be nice to utilize instead of creating an additional one.

Reply:
Create this tech debt ticket for refactoring https://folio-org.atlassian.net/browse/MODINVSTOR-1285